WordPress/Classes/ViewRelated/Post/AbstractPostListViewController.swift

```diff
@@ -695,7 +695,7 @@ class AbstractPostListViewController : UIViewController, WPContentSyncHelperDele
     func updateForLocalPostsMatchingSearchText() {
         resetTableViewContentOffset()
         updateAndPerformFetchRequest()
-        refreshCachedRowHeightsForTableViewWidth()
+        tableViewHandler.clearCachedRowHeights()
```

Comment:
For the most optimization, I think we could clear cached row heights before calling `updateAndPerformFetchRequest`.  When the fetch request is executed the table view will automatically refresh its content (yay NSFetchedResultsControllerDelegate) and refreshed row heights will then be cached again.   It should make the final call to `tableView.reloadDate` unnecessary.